src/features/requestedPackages/components/RequestedPackage.tsx

```diff
@@ -10,48 +13,59 @@ interface IRequestedPackageProps {
   isLast: boolean;
 }
 
+const StyledTableCell = styled(TableCell)(() => ({
+  width: 190
+}));
+
+const StyledTypography = styled(Typography)(() => ({
+  fontSize: "13px",
+  color: "#333"
+}));
+
 export const RequestedPackage = ({
   requestedPackage,
   isLast
 }: IRequestedPackageProps) => {
+  const { versionsWithoutConstraints, versionsWithConstraints } =
+    useAppSelector(state => state.requestedPackages);
```

Comment:
This is how its done in the [RequestPackagesTableRow](https://github.com/conda-incubator/conda-store-ui/blob/main/src/features/requestedPackages/components/RequestedPackagesTableRow.tsx#L31) component on the edit environment page. (Yes, it's confusing but there are nearly duplicate components for the edit versus view pages.)